RunningOrder/Shared/MainView+Logic.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MainView+Logic.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 19/02/2021.
+//  Copyright © 2021 Worldline. All rights reserved.
+//
+
+import SwiftUI
+import Combine
+
+extension MainView {
+    final class Logic: ObservableObject {
```

Comment:
class name is unusual, why did you call it Logic ?

Reply:
In terms of design pattern and Architeture, we should have a separation of concerns between the logic work / business rules, and the views. In a MVVM architecture, we usually have ViewModels that contains bindings towards the associated View, and the logic of the view. Here I did another cut : I separated just the logic part from the view, not the bindings that could leverage all the power of SwiftUI with property wrappers. This way we have just Logic unit linked to their views. I am open if you find a better name :)